Packs/PrismaCloud/ReleaseNotes/4_2_8.md

```diff
@@ -0,0 +1,31 @@
+
+#### Classifiers
+
+##### Prisma Cloud - Classifier
+
+- Removed the following policies from the classifier since they no longer exist on Prisma Cloud:
+  - AWS Security Group allows all traffic on FTP port (21)
+  - AWS Security Group allows all traffic on FTP port (21)
+  - AWS Security Group overly permissive to all traffic
+  - AWS Security Group allows all traffic on PostgreSQL port (5432)
+  - AWS Security Group allows all traffic on Telnet port (23)
+  - AWS Security Group allows all traffic on CIFS port (445)
+  - AWS Security Group allows all traffic on NetBIOS port (137)
+  - AWS Security Group Inbound rule overly permissive to all traffic on all protocols (-1)
+  - AWS Security Group allows all traffic on MYSQL port (3306)
+  - GCP Kubernetes Engine Clusters Basic Authentication is set to Enabled
+  - AWS Security Group allows all traffic on DNS port (53)
+  - AWS Security Group allows all traffic on SQL Server port (1433)
+  - AWS Security Group allows all traffic on VNC Server port (5900)
+  - AWS Security Group allows all traffic on VNC Listener port (5500)
+  - AWS Security Group allows all traffic on NetBIOS port (138)
+  - AWS Security Group allows all traffic on MSQL port (4333)
+  - AWS Security Group allows all traffic on Windows RPC port (135)
+  - AWS Security Group allows all traffic on SMTP port (25)
+  - Azure SQL Database with Threat Retention less than or equals to 90 days
+  - AWS Security Group allows all traffic on FTP-Data port (20)
+  - AWS Security Group allows all traffic on SQL Server port (1434)
+  - GCP Kubernetes Engine Clusters web UI/Dashboard is set to Enabled
```

Comment:
missing `AWS IAM password policy does not have a symbol`

Reply:
It's not missing. It's at the end of the list.